### PR TITLE
Added Database-level Roles section

### LINKED ISSUE #4078: To add DB user roles
## What type of issue is it? (Choose one - delete the others)
Missing documentation - Info to be added in regards to DB user 'roles'

## What article/section is this about?
Please refer to the thready below for details:
https://umbraco.slack.com/archives/C02JJQU9B/p1654609224615029 

## Describe the issue
Normally, a DB user is assigned `db_owner` role.

Other roles that can be assigned are as below:
`db_datareader` and `db_datawriter` for normal use.
`db_ddladmin` is needed for installs and upgrades (of the CMS and/or any packages that create database tables).

### PR DESCRIPTION
**Pull request by eshanrnh**
Fixes #4078

### CODE REVIEW

**Comment by AndyButland** [comment]
I don't think this is quite what's needed I'm afraid.  Whilst the information is correct, the reason this is useful to document is not really around creating custom database tables.  It's more what roles does the database account need to have that is used in the connection string for configuring Umbraco.

So it should probably go on [this page](https://our.umbraco.com/documentation/Fundamentals/Setup/Requirements/).

And state something along the lines of...

The database account used in the connection string will need permissions to read and write from tables, as well as create schema during installs and upgrades.

The `db_owner` has full permissions on the database and can be used.

If you prefer to use an account with more restricted permissions, `db_datareader` and `db_datawriter` will be needed for normal use.

With `db_ddladmin` required for installs and upgrades (of the CMS and/or any packages that create database tables).

By all means include your table explaining a bit more what these roles mean (or link to the Microsoft docs page where you got the details from).

**Comment by eshanrnh** [comment]
Thanks for your review, @AndyButland 😃 I've implemented the comments.

File: Fundamentals/Setup/Requirements/index.md [hunk 1]
```diff
@@ -50,3 +50,15 @@ You can use Umbraco Cloud to manage the hosting infrastructure. All Umbraco Clou
...
+
+## Database Account Roles
+
+The database account used in the connection string will need permissions to read and write from tables as well as create schema during installs and upgrades:
+
+* The `db_owner` role has full permissions on the database.
+
+* The `db_datareader` and `db_datawriter` roles will be needed for normal use to use an account with more restricted permissions.
+
+* The `db_ddladmin` role can perform installs and upgrades of the CMS and/or any packages that create database tables.
```
Comment (AndyButland) [inline Fundamentals/Setup/Requirements/index.md:62]:
```suggestion
* To use an account with more restricted permissions, the `db_datareader` and `db_datawriter` roles will be needed for normal use to read from and write to the database.  The `db_ddladmin` role, which can modify database schema, is required for installs and upgrades of the CMS and/or any packages that create database tables.
```

I suggest the above for this, as overall there are really two options - either you use `db_owner` (which can do everything)., or you use an account with only the necessary roles, which then are listed out.

**Comment by eshanrnh** [comment]
@AndyButland, I've accepted your suggestion and added the section to v7, v8, and v9 files as well.

**Review by AndyButland — APPROVED** [review]
Looks good, and good idea, as the same information applies for all versions.

**Pull request merged**